.size-snapshot.json

```diff
@@ -1,15 +1,15 @@
 {
   "dist/ui-kit.esm.js": {
-    "bundled": 707359,
-    "minified": 507454,
-    "gzipped": 97912,
+    "bundled": 866010,
+    "minified": 595824,
+    "gzipped": 121125,
     "treeshaked": {
       "rollup": {
-        "code": 354503,
-        "import_statements": 910
+        "code": 429358,
```

Comment:
this goes up, but hopefully it's a win due to not shipping two versions of emotion 🤷‍♂ 🤔

Reply:
Hmmm, but shouldn't that make the bundle size go down?

Reply:
I don't think size snapshot includes dependencies.

Reply:
Ah, alright, then it makes sense.

Reply:
I'm not 100% sure how size snapshot works to be honest. But I wouldn't hold back a dependency update just to save a few kb 🤷‍♂